python/deltalake/table.py

```diff
@@ -18,6 +18,27 @@ def version(self) -> int:
     def files(self) -> List[str]:
         return self._table.files()
 
+    def files_by_partitions(self, partition_filters: List[Tuple]) -> List[str]:
+        """
+        Partitions which do not match the filter predicate will be removed from scanned data.
+        Predicates are expressed in disjunctive normal form (DNF), like [("x", "=", "a"), ...].
+        DNF allows arbitrary boolean logical combinations of single partition predicates.
+        The innermost tuples each describe a single partition predicate.
+        The list of inner predicates is interpreted as a conjunction (AND), forming a more selective and multiple
+        partition predicates.
+        Each tuple has format: (key, op, value) and compares the key with the value.
+        The supported op are: =, !=, in and not in.
```

Comment:
```suggestion
        The supported op are: `=`, `!=`, `in`, and `not in`.
```

I found this comment a bit confusing since `and` is frequently used in predicate expressions, I think I've quoted the right things here :laughing:

Reply:
👍